cyclus/system.py

```diff
@@ -45,7 +45,7 @@ def websockets():
 
 @lazyobject
 def concurrent_futures():
```

Comment:
This should look like the curio replacement above, but replace curio with `concurrent_futures`.